### DISCUSSION TITLE
Pico W: Can you specify a  DCHP/IP Address Range for network.WLAN(network.AP_IF)?

### DISCUSSION THREAD

**Discussion by erzahurak** [micropython/micropython#15132] [RP2040 / Pico]
Howdy,

I'm trying to create a WIFI AP on a Pico W, which is easy enough, BUT...

I would like to specify/configure an IP Range/Mask from which connecting clients receive, rather than the default 192.168.4.16+.

Setting the network interface's ip/mask is simple, but that does get picked up by (what I'm presuming is) DHCP for clients.

Is it possible to set your own IP range?  If so, what am I missing?

(The more I think about this, there's really no practical reason to do this, so I guess this is just curiosity more than anything.)

Thanks!

Ed

**Comment by davefes** [comment]
Just an idea ... I noticed last time I was looking into my router menu that I could set  an IP range.  Maybe, that would work??

**Reply by erzahurak** [reply]
There isn't a router menu in this case -- I'm setting the Pico W up as the access point, rather than connecting *to* an access point with the Pico W; the other client(s) will be connecting to the Pico W.  (There's no routing, per se, either.  Communication will be limited to the Pico W and its wifi-connected clients.)

**Comment by davefes** [comment]
Right ... I am an AP beginner!

**Reply by erzahurak** [reply]
No worries, it's all good. :)
